docs/examples/irf_tool_config.json

```diff
@@ -24,8 +24,8 @@
     "max_theta_cut": 0.32,
     "fill_theta_cut": 0.32,
     "min_alpha_cut": 1,
-    "max_alpha_cut": 45,
-    "fill_alpha_cut": 45,
+    "max_alpha_cut": 20,
```

Comment:
These default values needs to be updated in `lstchain/io/event_selection` as well.